pkg/batcher/topology_builder.go

```diff
@@ -0,0 +1,103 @@
+package batcher
+
+import (
+	"github.com/StackVista/stackstate-agent/pkg/collector/check"
+	"github.com/StackVista/stackstate-agent/pkg/topology"
+)
+
+// Topologies is the type representing topologies gathered per check
+type Topologies map[check.ID]topology.Topology
+
+// TopologyBuilder is a helper class to build topology based on submitted data
```

Comment:
I would add comments related to the fact that it is not thread-safe

Reply:
done